packages/framework/presence/src/test/presenceManager.spec.ts

```diff
@@ -135,157 +195,199 @@ describe("Presence", () => {
+						const joinedAttendees = simulateAttendeeJoin([rejoinAttendeeSignalInfo]);
+						assert(
+							joinedAttendees.length === 1 && joinedAttendees[0] !== undefined,
+							"Expected exactly one attendee to be announced",
+						);
 
-						// Verify
-						verifyNewAttendee(rejoinAttendeeConnectionId);
+						verifyAttendee(joinedAttendees[0], rejoinAttendeeConnectionId);
 					});
 
 					it("as collateral and disconnected is NOT announced via `attendeeJoined`", () => {
 						// Setup - remove connections from audience
 						const collateralAttendeeConnectionId = "client3";
-						const collateralAttendeeSignal = generateBasicClientJoin(clock.now - 10, {
-							averageLatency: 40,
+						runtime.removeMember(initialAttendeeConnectionId);
+						runtime.removeMember(collateralAttendeeConnectionId);
+						const collateralAttendeeSignalInfo = {
 							clientSessionId: attendeeSessionId,
 							clientConnectionId: rejoinAttendeeConnectionId,
 							connectionOrder: 1,
+							fixedTime: clock.now - 10,
 							updateProviders: ["client2"],
 							priorClientToSessionId: {
-								...initialAttendeeSignal.content.data["system:presence"].clientToSessionId,
+								[initialAttendeeConnectionId]: {
+									rev: 0,
+									timestamp: initialAttendeeSignalInfo.fixedTime,
+									value: attendeeSessionId,
+								},
 								[collateralAttendeeConnectionId]: {
 									rev: 0,
 									timestamp: 0,
 									value: "collateral-id",
 								},
 							},
-						});
-						runtime.removeMember(initialAttendeeConnectionId);
-						runtime.removeMember(collateralAttendeeConnectionId);
+						};
 
 						// Act - simulate join message from client
-						presence.processSignal("", collateralAttendeeSignal, false);
+						const joinedAttendees = simulateAttendeeJoin([collateralAttendeeSignalInfo]);
 
-						// Verify only the rejoining attendee is announced
-						verifyNewAttendee(rejoinAttendeeConnectionId);
+						// Verify - only the rejoining attendee is announced
+						assert(
+							joinedAttendees.length === 1 && joinedAttendees[0] !== undefined,
+							"Expected exactly one attendee to be announced",
+						);
+
+						verifyAttendee(joinedAttendees[0], rejoinAttendeeConnectionId);
+					});
+
+					it.skip("is announced via `attendeeJoined` when a second joining attendee is unknown to audience", () => {
+						// SETUP - create attendee signals
+
+						// Signal for second joining attendee
+						const secondJoinSignalInfo = {
+							clientSessionId: "sessionId-9",
+							clientConnectionId: "client9", // Unknown to audience
+							fixedTime: clock.now - 40,
+							averageLatency: 10,
+							updateProviders: ["client2"], // Join response is requested from the initial joining attendee
+						};
+
+						// Join response signal broadcasted from initial joining attendee (responding to the second joining attendee message sent above)
+						const reponseSignalInfo = {
```

Comment:
two typos
- "broadcast" (is also past tense; no "ed")
- `responseSignalInfo` (missing "s")

Reply:
Reponse is french for response. Can't think of an excuse for broadcasted yet.